arch/arm/boot/dts/xilinx/ni-zynq.dtsi

```diff
@@ -140,6 +140,16 @@
 
 &smcc {
 	status = "okay";
+	/*
+	 * Bluefin devices require different memclk frequency
+	 * for different timing modes
+	 */
+	memclk-timing-frequency = <0 83333333>,
+							<1 166666666>,
+							<2 166666666>,
+							<3 166666666>,
+							<4 166666666>,
+							<5 166666666>;
```

Comment:
Indent misalignment most likely due to IDE set to show tab as 4 spaces. Let's stick to tab being 8 chars.
https://www.kernel.org/doc/html/latest/process/coding-style.html#indentation